source/postprocess/heat_flux_statistics.cc

```diff
@@ -128,8 +128,13 @@ namespace aspect
                       fe_face_values.JxW(q);
                   }
 
-                local_boundary_fluxes[cell->face(f)->boundary_indicator()]
-                += local_normal_flux;
+                const types::boundary_id bdry_indicator
```

Comment:
Are you willing to spell it out (`boundary_indicator` instead of `bdry_indicator`)?